<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.251">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Leila Elabbady, Sharmishtaa Seshamani, Shang Mu, Gayathri Mahalingam, Casey Schneider-Mizell, Agnes Bodor, J. Alexander Bae, Derrick Brittain, JoAnn Buchanan, Daniel J. Bumbarger, Manuel A. Castro, Erick Cobos, Sven Dorkenwald, Paul G. Fahey, Emmanouil Froudarakis, Akhilesh Halageri, Zhen Jia, Chris Jordan, Dan Kapner, Nico Kemnitz, Sam Kinn, Kisuk Lee, Kai Li, Ran Lu, Thomas Macrina, Eric Mitchell, Shanka Subhra Mondal, Barak Nehoran, Stelios Papadopoulos, Saumil Patel, Xaq Pitkow, Sergiy Popovych, Jacob Reimer, William Silversmith, Fabian H. Sinz, Marc Takeno, Russel Torres, Nicholas Turner, William Wong, Jingpeng Wu, Wenjing Yin, Szi-chieh Yu, Andreas Tolias, H. Sebastian Seung, R. Clay Reid, Nuno Maçarico Da Costa, Forrest Collman">

<title>Quantitative Census of Local Somatic Features in Mouse Visual Cortex</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


</head>

<body>

<div id="quarto-search-results"></div>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#overview" id="toc-overview" class="nav-link active" data-scroll-target="#overview">Overview</a></li>
  <li><a href="#abstract" id="toc-abstract" class="nav-link" data-scroll-target="#abstract">Abstract</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion">Discussion</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods">Methods</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Quantitative Census of Local Somatic Features in Mouse Visual Cortex</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Leila Elabbady, Sharmishtaa Seshamani, Shang Mu, Gayathri Mahalingam, Casey Schneider-Mizell, Agnes Bodor, J. Alexander Bae, Derrick Brittain, JoAnn Buchanan, Daniel J. Bumbarger, Manuel A. Castro, Erick Cobos, Sven Dorkenwald, Paul G. Fahey, Emmanouil Froudarakis, Akhilesh Halageri, Zhen Jia, Chris Jordan, Dan Kapner, Nico Kemnitz, Sam Kinn, Kisuk Lee, Kai Li, Ran Lu, Thomas Macrina, Eric Mitchell, Shanka Subhra Mondal, Barak Nehoran, Stelios Papadopoulos, Saumil Patel, Xaq Pitkow, Sergiy Popovych, Jacob Reimer, William Silversmith, Fabian H. Sinz, Marc Takeno, Russel Torres, Nicholas Turner, William Wong, Jingpeng Wu, Wenjing Yin, Szi-chieh Yu, Andreas Tolias, H. Sebastian Seung, R. Clay Reid, Nuno Maçarico Da Costa, Forrest Collman </p>
          </div>
  </div>
    
    
  </div>
  

</header>

<section id="overview" class="level3">
<h3 class="anchored" data-anchor-id="overview">Overview</h3>
<p>Paper Selection: A recent paper from my PhD work currently available on biorxiv: https://doi.org/10.1101/2022.07.20.499976</p>
<p>Motivation: When discussing how scientists tend to read papers, a common response is that people focus on the figures jumping from one to the next and largely skipping the text. While I have tried this approach I find that I attempt to focus on the figures but always look for what the authors acutally said about the figure. With this in mind, I thought wouldn’t it be nice if we could see all the excerpts where this figure has been referenced in the paper, similar to how you can see where a function/variable has been referenced when programming?</p>
<p>Augmentation: Adding an expandable callout window above each figure that allows you to see all the points where the figure was referenced without having to navigate away from the figure itself.</p>
<p>Future changes could include adding the ability for the reader to jump to the exact section of a given excerpt if they want to read more. Adding a scroll effect to the content in the callout box so that the figure is always visible.</p>
</section>
<section id="abstract" class="level3">
<h3 class="anchored" data-anchor-id="abstract">Abstract</h3>
<p>Mammalian neocortex contains a highly diverse set of cell types. These types have been mapped systematically using a variety of molecular, electrophysiological and morphological approaches. Each modality offers new perspectives on the variation of biological processes underlying cell type specialization. While many morphological surveys focus on branching patterns of individual cells, fewer have been devoted to sub-cellular structure of cells. Electron microscopy (EM) provides dense ultrastructural examination and an unbiased perspective into the subcellular organization of brain cells, including their synaptic connectivity and nanometer scale morphology. Here we present the first systematic survey of the somatic region of nearly 100,000 cortical cells, using quantitative features obtained from EM. This analysis demonstrates a surprising sufficiency of the perisomatic region to recapitulate many known aspects of cortical organization, while also revealing novel relationships. Parameters of cell size, nuclear infolding and somatic synaptic innervation co-vary with distinct patterns across depth and between types. Further, we describe how these subcellular features can be used to create highly accurate predictions of cell-types across large scale EM datasets. More generally, our results suggest that the shifts in cellular physiology and molecular programming seen across cell types accompany profound differences in the fine-scale structure of cells.</p>
</section>
<section id="introduction" class="level3">
<h3 class="anchored" data-anchor-id="introduction">Introduction</h3>
<p>The different cell types of neocortex vary in their complex dendritic and axonal morphologies,<span class="citation" data-cites="oberlaender_cell_2012">Oberlaender et al. (<a href="#ref-oberlaender_cell_2012" role="doc-biblioref">2012</a>)</span>,<span class="citation" data-cites="kanari_objective_2019">Kanari et al. (<a href="#ref-kanari_objective_2019" role="doc-biblioref">2019</a>)</span>,<span class="citation" data-cites="delgado-escueta_fine_1992">Delgado-Escueta (<a href="#ref-delgado-escueta_fine_1992" role="doc-biblioref">1992</a>)</span>,<span class="citation" data-cites="wang_anatomical_2004">Wang et al. (<a href="#ref-wang_anatomical_2004" role="doc-biblioref">2004</a>)</span>,<span class="citation" data-cites="martin_form_1984">Martin and Whitteridge (<a href="#ref-martin_form_1984" role="doc-biblioref">1984</a>)</span>,<span class="citation" data-cites="gilbert_morphology_1979">Gilbert and Wiesel (<a href="#ref-gilbert_morphology_1979" role="doc-biblioref">1979</a>)</span>,<span class="citation" data-cites="markram_reconstruction_2015">Markram et al. (<a href="#ref-markram_reconstruction_2015" role="doc-biblioref">2015</a>)</span> along with their physiological properties <span class="citation" data-cites="gouwens_classification_2019">Gouwens et al. (<a href="#ref-gouwens_classification_2019" role="doc-biblioref">2019</a>)</span>,<span class="citation" data-cites="scala_phenotypic_2021">Scala et al. (<a href="#ref-scala_phenotypic_2021" role="doc-biblioref">2021</a>)</span> and molecular expression profiles.@tasic_shared_2018,<span class="citation" data-cites="zeisel_cell_2015">Zeisel et al. (<a href="#ref-zeisel_cell_2015" role="doc-biblioref">2015</a>)</span>,<span class="citation" data-cites="jones_varieties_1975">Jones (<a href="#ref-jones_varieties_1975" role="doc-biblioref">1975</a>)</span>,<span class="citation" data-cites="ascoli_petilla_2008">Ascoli et al. (<a href="#ref-ascoli_petilla_2008" role="doc-biblioref">2008</a>)</span> Large scale electron microscopy (EM) data with automated reconstructions <span class="citation" data-cites="consortium_functional_2021">Consortium et al. (<a href="#ref-consortium_functional_2021" role="doc-biblioref">2021</a>)</span>,<span class="citation" data-cites="macrina_petascale_2021">Macrina et al. (<a href="#ref-macrina_petascale_2021" role="doc-biblioref">2021</a>)</span>,<span class="citation" data-cites="shapson-coe_connectomic_2021">Shapson-Coe et al. (<a href="#ref-shapson-coe_connectomic_2021" role="doc-biblioref">2021</a>)</span>,<span class="citation" data-cites="motta_dense_2019">Motta et al. (<a href="#ref-motta_dense_2019" role="doc-biblioref">2019</a>)</span> creates both opportunities and challenges for examining the structural variation of cell types across cortex. EM provides an opportunity to study a host of morphological and subcellular detail that has not historically been studied at a large quantitative scale. Further, large scale EM enables the study of connectivity patterns between diverse cell types. However, in very large scale EM datasets with tens to hundreds of thousands of cells, it is impractical to manually classify all the cells in the dataset. Existing methods for automated cell-typing based on morphology often necessitate nearly complete axonal or dendritic reconstructions. <span class="citation" data-cites="kanari_objective_2019">Kanari et al. (<a href="#ref-kanari_objective_2019" role="doc-biblioref">2019</a>)</span>,<span class="citation" data-cites="gouwens_classification_2019">Gouwens et al. (<a href="#ref-gouwens_classification_2019" role="doc-biblioref">2019</a>)</span>,<span class="citation" data-cites="scorcioni_l-measure_2008">Scorcioni, Polavaram, and Ascoli (<a href="#ref-scorcioni_l-measure_2008" role="doc-biblioref">2008</a>)</span> Such reconstructions currently require manual correction to the segmentation, often referred to as proofreading, which is prohibitively time consuming at scale. Furthermore, cells can be classified at varying degrees of refinement, from high level distinctions (neurons vs non-neuronal cells), to highly specific types (Chandelier cells, or a particular transcriptomic type). Depending on the scientific question and perspective of the investigator, different levels of cell type classification are desirable. This highlights the need for a flexible automated solution which can map differing cell classification schemes at varying levels of resolution.</p>
<p>Focusing on the somatic region of cells provides a unique window for analysis that balances these challenges and opportunities. The probability of a segmentation containing an error grows with the size of the segmentation, so analysis that is restricted to a local region is less sensitive to such errors. The soma is a unique local region in that there is only one per cell, and of obvious importance to the biology of the cell. For example, because the soma is the hub for dendritic branches and sits close to the site of action potential initiation, inputs to this region have a great influence on the firing of the cell. Moreover, all the machinery for translating DNA into proteins sits within this cellular compartment, and structural correlates of this machinery are all visible within the EM imagery. Despite this, several basic properties of this compartment have not been measured quantitatively and at a large scale and may vary between types. First, individual cells have widely varying sizes and corresponding demands for gene expression and metabolic load,<span class="citation" data-cites="tasic_shared_2018">Tasic et al. (<a href="#ref-tasic_shared_2018" role="doc-biblioref">2018</a>)</span> which might induce shifts in their nuclear and cytosolic size. Second, there are anecdotal reports of differences in nucleus infoldings varying across cortical cell types and modulated by activity. <span class="citation" data-cites="wittmann_synaptic_2009">Wittmann et al. (<a href="#ref-wittmann_synaptic_2009" role="doc-biblioref">2009</a>)</span>,<span class="citation" data-cites="peters_neuronal_1985">Peters and Kara (<a href="#ref-peters_neuronal_1985" role="doc-biblioref">1985</a>)</span> Third, in excitatory cells, somatic input has been noted to be dominated by inhibitory sources.@villa_excitatory_2016 Although this is less well characterized for inhibitory subtypes, some have larger fractions of excitatory input.@davis_microcircuitry_1979,<span class="citation" data-cites="hwang_3d_2021">Hwang et al. (<a href="#ref-hwang_3d_2021" role="doc-biblioref">2021</a>)</span> In addition, different types of inhibitory axons show preference or avoidance of the somatic compartment.@villa_excitatory_2016,<span class="citation" data-cites="davis_microcircuitry_1979">Davis and Sterling (<a href="#ref-davis_microcircuitry_1979" role="doc-biblioref">1979</a>)</span> Therefore, even a coarse measurement such as the total somatic synaptic density might reflect differences in cell-type specific connectivity <span class="citation" data-cites="hwang_3d_2021">Hwang et al. (<a href="#ref-hwang_3d_2021" role="doc-biblioref">2021</a>)</span>,<span class="citation" data-cites="campagnola_local_2022">Campagnola et al. (<a href="#ref-campagnola_local_2022" role="doc-biblioref">2022</a>)</span>,<span class="citation" data-cites="hage_synaptic_2022">Hage et al. (<a href="#ref-hage_synaptic_2022" role="doc-biblioref">2022</a>)</span> and thus vary across types. Fourth, smaller scale features of cellular morphology have been described for specific cell types,<span class="citation" data-cites="hwang_3d_2021">Hwang et al. (<a href="#ref-hwang_3d_2021" role="doc-biblioref">2021</a>)</span> but little quantitative work has been done in this area across all the full diversity of cortical cells. The local somatic region contains both somatic membrane and proximal dendritic shafts thereby enabling the examination of properties like the size and distribution of spines.@wang_anatomical_2004,<span class="citation" data-cites="georgiou_subpopulation_2022">Georgiou et al. (<a href="#ref-georgiou_subpopulation_2022" role="doc-biblioref">2022</a>)</span></p>
<p>Here we report the first large scale survey of somatic properties of cells as measured with EM across the cortical depth. From the analysis of these properties we describe some novel structural patterns of cortical organization. We integrate our analysis with a systematic anatomical study of a columnar region of cortex, where an anatomical based classification of all cells has been performed. The concept of a cell type is a categorical label which implies that all the cells within that category have stereotyped patterns to all its relevant attributes. There is not broad agreement about what attributes are relevant or what methods define stereotypy, and so cell type definitions can be controversial. We use the term “cell type” in this manuscript to refer to the abstract concept of stereotyped variation across cells. However, when talking about particular categories, we will use the term “class” and “subclass” here to refer to levels of distinctions within this particular categorical scheme. This classification describes cell classes at the level of neurons and non-neuronal classes, and sub-classes within the excitatory and inhibitory populations. We demonstrate that there is enough variation in features amongst cells to build quantitative cellular classification models that predict cell classes and subclasses with high accuracy and enable dataset-wide predictions. This both improves the ability to find many examples of cells of a specific category and allows better measurements of neuronal connectivity with less proofreading (in preparation), solving one of the key problems to exploring large scale EM datasets. These results demonstrate that cortical cells have profound differences in the anatomical structure of their somatic region and provides a novel platform to understand how specialized biological demands affect somatic and nuclear properties.</p>
</section>
<section id="results" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="results">Results</h3>
<p>We analyzed the larger portion of a 1.4mm x 800um x 800um volumetric serial section EM dataset from mouse visual cortex, <span class="citation" data-cites="consortium_functional_2021">Consortium et al. (<a href="#ref-consortium_functional_2021" role="doc-biblioref">2021</a>)</span> that contains a dense segmentation of cells along with a nucleus segmentation and large scale synapse detection. <span class="citation" data-cites="macrina_petascale_2021">Macrina et al. (<a href="#ref-macrina_petascale_2021" role="doc-biblioref">2021</a>)</span>,<span class="citation" data-cites="turner_reconstruction_2022">Turner et al. (<a href="#ref-turner_reconstruction_2022" role="doc-biblioref">2022</a>)</span> The quality of the automated segmentation and amount of proofreading varies across the dataset and different structures, but in general the quality of the nucleus segmentation is high for cells away from the edges of the dataset, and the quality of the cellular segmentation is high near the somas of cells. This dataset includes 86,916,454 high quality nuclear detections fully enclosed within the boundaries of the volume (see methods) and spans cortical Layer 1 through to the white matter (Fig. 1a). For each cell we analyzed the nucleus and somatic region of the 3D segmentation (Fig. 1b) within a 15um radius around the nuclear center point. We also included synapse information within this region (Fig. 1b,c-e). For a subset of 1909 cells in a 100 um columnar region of VISp (referred here to as the column), we had manual labels for cellular classes and neuronal subclasses (Excitatory: Layers 2/3,4,6, and Layer 5 inter-telencenphalic (IT), near-projecting (NP) and extra-telencenphalic (ET) and Inhibitory: Martinotti cell (MC), Basket cell (BC), Bi-polar cell (BPC) and neurogliaform cell (NGC), Nonneurons: astrocyte, oligo-precursor cell (OPC), oligodendrocyte, microglia, pericyte) (Fig. 1b). For all neurons within this column, we also analyzed the nano-scale structure of the postsynaptic compartments, what we are terming a “post synaptic shape”(PSS) (see methods) within 60um of the nucleus center (Fig. 1g).</p>
<div class="callout-note callout callout-style-default no-icon callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Text References to Figure 1
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>This dataset includes 86,916,454 high quality nuclear detections fully enclosed within the boundaries of the volume (see methods) and spans cortical Layer 1 through to the white matter (Fig. 1a).</p>
<p>For each cell we analyzed the nucleus and somatic region of the 3D segmentation within a 15um radius around the nuclear center point. We also included synapse information within this region (Fig. 1b,c-e).</p>
<p>For a subset of 1909 cells in a 100 um columnar region of VISp (referred here to as the column), we had manual labels for cellular classes and neuronal subclasses (Excitatory: Layers 2/3,4,6, and Layer 5 inter-telencenphalic (IT), near-projecting (NP) and extra-telencenphalic (ET) and Inhibitory: Martinotti cell (MC), Basket cell (BC), Bi-polar cell (BPC) and neurogliaform cell (NGC), Nonneurons: astrocyte, oligo-precursor cell (OPC), oligodendrocyte, microglia, pericyte) (Fig. 1b).</p>
<p>For all neurons within this column, we also analyzed the nano-scale structure of the postsynaptic compartments, what we are terming a “post synaptic shape”(PSS) (see methods) within 60um of the nucleus center (Fig. 1g).</p>
</div>
</div>
</div>
<div id="fig-Basil_Minnie_Dataset_Figure1" class="quarto-figure quarto-figure-center column-page anchored">
<figure class="figure">
<p><img src="Basil_Minnie_Dataset_Figure1.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;1: Systematic evaluation of soma and nucleus features from a large scale EM dataset. a) Rendering of a small fraction of neurons from a large scale dataset covering all layers of cortex and multiple visual areas, with 1207 rendered and then cutaway to reveal the full morphology of 2 selected neurons on the right portion of the dataset. b) rendering of all the neuronal cells from a 100 um column of data for which manual cell type labels were given. Cell class and subclass labels and color schemes used in the paper are shown to the right. c) a 2d view of a single EM section of data with nuclear detections in light blue and somatic regions in gray. d) an enlarged view of the portion of c (red box) showing a single somatic segmentation with the nucleus in blue and the cytoplasm of the cell in gray. e) a series of enlarged 2d views of a single synapse onto the somatic region in d (orange box) showing a single synapse onto that cell. The left column shows the raw EM image, and the left column includes the synapse prediction overlaid in orange f) a 3d mesh rendering of the somatic region of a cell including its cellular membrane (gray), its nucleus segmentation (blue), and the location of synapses as orange spheres (size indicates voxel size of synapse prediction). g) Examples of fine scale morphologies of postsynaptic shapes.</figcaption><p></p>
</figure>
</div>
<section id="nucleus-features" class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="nucleus-features">Nucleus Features</h4>
<p>We began by systematically analyzing geometric properties of the nucleus, including their volume, surface area, and depth from the pial surface. In addition, we quantified the fraction of nucleus membrane area that was within an infolding (Fig. 2a-c) (see methods). The nucleus has anecdotally been described as having different degrees of infolding in different cell types,<span class="citation" data-cites="peters_neuronal_1985">Peters and Kara (<a href="#ref-peters_neuronal_1985" role="doc-biblioref">1985</a>)</span>, <span class="citation" data-cites="jorgens_deep_2017">Jorgens et al. (<a href="#ref-jorgens_deep_2017" role="doc-biblioref">2017</a>)</span> but a systematic quantification has not been done across cortical types.</p>
<p>Nucleus features of excitatory neurons have a striking laminar organization, wherein the borders between layer ⅔ (L23), layer 4 (L4), layer 5 (L5), and layer 6 (L6) are all clearly demarcated by shifts in the distribution of nucleus volumes. (Fig. 2d). Furthermore, the fraction of membrane inside an infolding also varies widely and systematically depending on depth (Fig. 2b). Layer ⅔ neurons have largely smooth nuclear membranes. There is a clear gradient of infolding within layer 4. All layer 5 excitatory cells have high degrees of infolding, despite the notable diversity of cell types and sizes within that population, which is reflected in the notable increased variation of nucleus volume in that layer. <span class="citation" data-cites="kim_three_2015">E. J. Kim et al. (<a href="#ref-kim_three_2015" role="doc-biblioref">2015</a>)</span> Infolding drops sharply again in layer 6 (Fig. 2c).</p>
<p>Inhibitory cells on the other hand have less striking laminar patterns, but with more overall variation. They have a wider variation of nucleus volumes, which are overlapping with excitatory cells, with the exception of the larger layer 5 neurons. They have 15 to 30% of their membrane within an infolding, regardless of their position within cortex. This makes them quite distinct from excitatory neurons in layer 1, 2/3, 4 and 6 of cortex, but highly overlapping in layer 5 (Fig. 2b).</p>
<p>Non-neuronal cells had generally smaller nuclei compared to neuronal cells, though astrocytes overlap in this distribution with the smallest neurons. Each non-neuronal cell class exhibited a distinct range and consistency in their nucleus volume across the layers of cortex. Non-neuronal cells generally did not have infoldings, though microglia, OPCs and oligodendrocytes had less spherical and convex shapes. Perictyes had the least reliable nucleus segmentations in the dataset, but when segmented well they had the smallest overall volumes with shapes dominated by their close apposition to the vascular walls (Fig. 2c).</p>
<p>Nucleus features were extracted for all cells in the dataset and a 2-dimensional embedding was computed. This low-dimensional space is plotted in Figure 2e with manual class labels in color and all the unlabelled cells in gray (column =1850, unlabelled=85,066)). The distinct clustering of different cell classes suggests that neuronal and non-neuronal cells can be separated using these metrics alone, and that most non-neuronal classes have distinct nuclear shapes (Fig. 2c).</p>
<div class="callout-note callout callout-style-default no-icon callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Text References to Figure 2
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>We began by systematically analyzing geometric properties of the nucleus, including their volume, surface area, and depth from the pial surface. In addition, we quantified the fraction of nucleus membrane area that was within an infolding (Fig. 2a-c)</p>
<p>Nucleus features of excitatory neurons have a striking laminar organization, wherein the borders between layer ⅔ (L23), layer 4 (L4), layer 5 (L5), and layer 6 (L6) are all clearly demarcated by shifts in the distribution of nucleus volumes. (Fig. 2d).</p>
<p>Furthermore, the fraction of membrane inside an infolding also varies widely and systematically depending on depth (Fig. 2b).</p>
<p>All layer 5 excitatory cells have high degrees of infolding, despite the notable diversity of cell types and sizes within that population, which is reflected in the notable increased variation of nucleus volume in that layer. <span class="citation" data-cites="kim_three_2015">E. J. Kim et al. (<a href="#ref-kim_three_2015" role="doc-biblioref">2015</a>)</span> Infolding drops sharply again in layer 6 (Fig. 2c).</p>
<p>This makes them quite distinct from excitatory neurons in layer 1, 2/3, 4 and 6 of cortex, but highly overlapping in layer 5 (Fig. 2b).</p>
<p>Perictyes had the least reliable nucleus segmentations in the dataset, but when segmented well they had the smallest overall volumes with shapes dominated by their close apposition to the vascular walls (Fig. 2c).</p>
<p>The distinct clustering of different cell classes suggests that neuronal and non-neuronal cells can be separated using these metrics alone, and that most non-neuronal classes have distinct nuclear shapes (Fig. 2c).</p>
<p>In dataset 1, the degree of nuclear infoldings varied starkly by excitatory sub-class with different patterns of variation within laminar layers. Specifically, we report the existence of a spatial gradient within excitatory cells in layer 4, which might reflect previously described gradients in gene expression. <span class="citation" data-cites="tasic_shared_2018">Tasic et al. (<a href="#ref-tasic_shared_2018" role="doc-biblioref">2018</a>)</span></p>
<p>Alternatively, the overall pattern of nuclear infolding which is shared across layer 5 types, might suggest a common mechanism despite their broad differences in gene expression, electrophysiological properties and connectivity. <span class="citation" data-cites="gouwens_classification_2019">Gouwens et al. (<a href="#ref-gouwens_classification_2019" role="doc-biblioref">2019</a>)</span>,<span class="citation" data-cites="tasic_shared_2018">Tasic et al. (<a href="#ref-tasic_shared_2018" role="doc-biblioref">2018</a>)</span></p>
<p>There is some evidence that in some cell types that nucleus infoldings are distinct calcium microdomains. <span class="citation" data-cites="turner_reconstruction_2022">Turner et al. (<a href="#ref-turner_reconstruction_2022" role="doc-biblioref">2022</a>)</span></p>
<p>A mechanism of infolding has been proposed whereby laminar associated domains (LADs) on the heterochromatin attach to the nuclear envelope, and a force is exerted that brings the heterochromatin under tension which causes the membrane to fold. <span class="citation" data-cites="schoen_nanoscale_2017">Schoen et al. (<a href="#ref-schoen_nanoscale_2017" role="doc-biblioref">2017</a>)</span></p>
</div>
</div>
</div>
<div class="column-page">
<p><img src="Basil_Minnie_Nuc_Figure2.png" class="img-fluid"></p>
<p>Nuclear shape and infolding features show laminar and cell-type distinctions. a) 2D view of a single EM section of data with the cell body in lightgray, the nuclear membrane in black with the portion of the membrane considered in a fold highlighted in orange. b) Fraction of the nuclear membrane within a fold plotted against distance from the pial surface. Human labeled cell-types are represented in color (n=1850) and unlabelled examples in light gray (n=85,066). Cortical layer boundaries are noted by the dashed lines. c) 3D mesh renderings of representative nuclei from different neuronal and nonneuronal cell-types with the folded surface area highlighted in orange. d) Nuclear volume in um3 plotted against distance from the pial surface, colored as in b). e) 2D UMAP embedding of all neuronal and nonneuronal cells inferred from nuclear size features, infolding features, and cortical depth (n=86,916).</p>
</div>
</section>
<section id="somatic-features" class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="somatic-features">Somatic Features</h4>
<p>We then calculated similar geometric properties of the somatic region of cells (see methods), including the total volume, surface area, cytosolic volume, the ratio of the nucleus volume to the soma volume, and the average distance from the centroid of the nucleus to the centroid of the soma. We also measured the number and density of synapses detected on the somatic region of the cell. Together these somatic and nucleus features represent a feature space that can be extracted automatically from most cells in the segmentation (75% of nuclei). We filtered out soma regions that represented direct soma to soma merges (n=3,852), which naturally fall out as statistical outliers (see methods, Extended Data figure 1a). Plotting just the metrics of the nucleus volume versus the surface area of the cell within 15 microns of the center of the nucleus, reveals a surprisingly striking separation between the major cell classes found in the brain (Fig. 3a).</p>
<p>Similar to their nucleus features, excitatory neurons showed laminar shifts in their somatic volume and surface area (Extended Data Fig. 5) The excitatory neurons show a consistent synapse density that slightly varies in a linear fashion with depth through the cortical volume, with a notable increase in variation in layer 5 that correlates with the 3 sub-classes found there with ET cells having larger synapse densities, NP cells with low synapse densities and IT cells in between (Fig. 3b, d).</p>
<p>Neurons identified as inhibitory showed less laminar variation in somatic size, however they are on average smaller than excitatory cells in the same cortical layer (Extended Data Fig. 5). On the other hand, inhibitory cells have much larger density of somatic innervation than excitatory cells, but also have a much wider degree of variation, reflecting their rich diversity of sub-classes (Fig. 3b,d).</p>
<p>Based on nucleus volume and soma cutout area alone (Fig. 3a), microglia, oligodendrocytes, OPCs, astrocytes, and pericytes occupy largely distinct portions of this 2-dimensional space. The notably large surface area measurement for astrocytes is explained by the high density of their processes near the soma. Moreover, the high prevalence of segmentation mergers of pericytes with cortical vasculature results in variability in their soma size features as represented by the range in soma surface area in Figure 3a. As expected, all nonneuronal cells had very low soma synapse counts and thus are clearly distinct from neurons across laminar layers (Fig. 3b).</p>
<p>Following the same framework as in Figure 2d, soma features were extracted for all cells in the dataset. A new low dimensional embedding was computed based on somatic features in addition to the previously reported nucleus features. Consistent with the diversity observed in individual features, low dimensional embeddings of the feature space show variation that correlates with the manually labeled cell classes (Fig. 3c). As before, nonneuronal cell classes occupy distinct areas of the feature space. Note that despite the variability in pericytic somatic features, they are still well separated in the low dimensional embedding space, likely due to the distinguishability from nuclear features alone. On the other hand, adding the soma features improves the separation between inhibitory and excitatory cells, with two distinctly inhibitory clusters and finer separation between excitatory neurons in different layers. However, some distinctions between neuronal classes are not as clear. For example, a subset of inhibitory cells overlap with the Layer 5 excitatory cells due to similarities in somatic size and fraction of nuclear infolding. We therefore searched for additional quantitative features that we could extract from the local region around cell bodies to aid in separating these neuronal classes further.</p>
<div class="callout-note callout callout-style-default no-icon callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Text References to Figure 3
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Plotting just the metrics of the nucleus volume versus the surface area of the cell within 15 microns of the center of the nucleus, reveals a surprisingly striking separation between the major cell classes found in the brain (Fig. 3a).</p>
<p>The excitatory neurons show a consistent synapse density that slightly varies in a linear fashion with depth through the cortical volume, with a notable increase in variation in layer 5 that correlates with the 3 sub-classes found there with ET cells having larger synapse densities, NP cells with low synapse densities and IT cells in between (Fig. 3b, d).</p>
<p>On the other hand, inhibitory cells have much larger density of somatic innervation than excitatory cells, but also have a much wider degree of variation, reflecting their rich diversity of sub-classes (Fig. 3b,d).</p>
<p>Based on nucleus volume and soma cutout area alone (Fig. 3a), microglia, oligodendrocytes, OPCs, astrocytes, and pericytes occupy largely distinct portions of this 2-dimensional space.</p>
<p>Moreover, the high prevalence of segmentation mergers of pericytes with cortical vasculature results in variability in their soma size features as represented by the range in soma surface area in Figure 3a.</p>
<p>As expected, all nonneuronal cells had very low soma synapse counts and thus are clearly distinct from neurons across laminar layers (Fig. 3b).</p>
<p>A new low dimensional embedding was computed based on somatic features in addition to the previously reported nucleus features. Consistent with the diversity observed in individual features, low dimensional embeddings of the feature space show variation that correlates with the manually labeled cell classes (Fig. 3c).</p>
</div>
</div>
</div>
<div class="column-page">
<p><img src="Basil_Minnie_Soma_Figure3.png" class="img-fluid"></p>
<p>Somatic shape and synapse features further distinguish neuronal and nonneuronal cell class types. a) Somatic surface area in um2 (within 15um from the nuclear center) plotted against nuclear volume um3. b) Somatic synapse density um2 plotted by distance from the pial surface. c) 2D UMAP embedding of all neuronal and nonneuronal cells inferred from somatic features, nuclear features and cortical depth. a-c) Human labeled class types in color and unlabelled cells in gray. d) 3D mesh renderings examples of 15 um somatic cutouts in gray and somatic synapses in orange. Size of individual spheres relative to the size of the post-synaptic density.</p>
</div>
</section>
<section id="post-synaptic-shape-features" class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="post-synaptic-shape-features">Post Synaptic Shape Features</h4>
<p>Proximal processes of cells display large variations in fine morphology. For example, some excitatory dendritic branches are covered with a high density of short stubby spines, others have long wispy spines or filopodia. Inhibitory branches vary from being very smooth and uniform, to those that vary in caliper, and some are covered in smaller spine-like protrusions. <span class="citation" data-cites="hwang_3d_2021">Hwang et al. (<a href="#ref-hwang_3d_2021" role="doc-biblioref">2021</a>)</span>,<span class="citation" data-cites="pchitskaya_dendritic_2020">Pchitskaya and Bezprozvanny (<a href="#ref-pchitskaya_dendritic_2020" role="doc-biblioref">2020</a>)</span> These differences are visible on the dendritic arbor even in close proximity to the soma (Fig. 4). We developed a method to summarize the statistics of the proximal arbor within 60 microns of the nucleus center. This feature combines the patterns of synaptic innervation with the fine morphological structure of the post-synaptic neuron surrounding any given synapse (Fig. 4). <span class="citation" data-cites="pchitskaya_dendritic_2020">Pchitskaya and Bezprozvanny (<a href="#ref-pchitskaya_dendritic_2020" role="doc-biblioref">2020</a>)</span> Briefly, this method computationally segments the compartment on the post-synaptic side of a synapse, which we refer to as the post-synaptic shape (PSS). This shape typically represents either a portion of the soma, the shaft of a dendrite, or a spiney protrusion, though it can also be onto an axon or axon-initial segment. We then used a PointNet auto-encoder to summarize a latent space of PSSs (Fig. 4a) (see methods). This resulted in a continuous latent space where PSS objects of similar morphological character were closer together (Extended Data Fig. 2). This representation recapitulates well known distinctions between PSSs such as dendritic shafts versus spines versus somatic compartments, but also more fine grained distinctions, such as variations in the size and shape of spines (Extended Data Fig. 3).</p>
<p>We then summarized a cell’s distribution of PSSs by binning the PSS space into 30 shape bins and 4 distance bins based upon the distance from the soma (see methods), resulting in a 120 dimensional vector, similar in spirit to a multi-dimensional Sholl analysis. <span class="citation" data-cites="sholl_dendritic_1953">Sholl (<a href="#ref-sholl_dendritic_1953" role="doc-biblioref">1953</a>)</span> Examining individual example cells of different types suggest that this feature effectively quantifies differences in the fine scale synaptic distribution of different cell types (Fig. 4c). For example, the layer 5 ET and basket cell both contain large numbers of somatic synapses, but the layer 5 ET cell has a much higher density of spines. The Layer 5 NP cell has a lower number of spines, but also the distribution of its spines is less concentrated in the PSS shape bins that correspond to smaller shorter spines, compared to the L5 ET cell. Some inhibitory cells contain counts in the bins associated with spines, consistent with the fact that some inhibitory dendrites do have protrusions. However, these protrusions most often lack the classic spine neck and bulbous head morphology of most spines on excitatory cells. Because they are similar in shape, this analysis groups these inhibitory “thorn” like protrusions with the smaller stubby spines found on excitatory cells. <span class="citation" data-cites="hwang_3d_2021">Hwang et al. (<a href="#ref-hwang_3d_2021" role="doc-biblioref">2021</a>)</span> Consistent with that, the bipolar example shown here contains an elevated number of such small protrusions, <span class="citation" data-cites="georgiou_subpopulation_2022">Georgiou et al. (<a href="#ref-georgiou_subpopulation_2022" role="doc-biblioref">2022</a>)</span> including in the somatic region, whereas the smoother dendritic shafts of this basket cell example contain virtually no counts in these bins. (Fig. 4c). This distance dependent PSS histogram was calculated for all neurons within the column. We then recomputed a low dimensional embedding by aggregating nucleus, somatic and PSS features (Fig. 4d). Note that the addition of the PSS-based features results in a near complete distinction between inhibitory and excitatory neurons within this space.</p>
<div class="callout-note callout callout-style-default no-icon callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-4-contents" aria-controls="callout-4" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Text References to Figure 4
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-4" class="callout-4-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>These differences are visible on the dendritic arbor even in close proximity to the soma (Fig. 4). We developed a method to summarize the statistics of the proximal arbor within 60 microns of the nucleus center.</p>
<p>This feature combines the patterns of synaptic innervation with the fine morphological structure of the post-synaptic neuron surrounding any given synapse (Fig. 4).</p>
<p>We then used a PointNet auto-encoder to summarize a latent space of PSSs (Fig. 4a) (see methods).</p>
<p>Examining individual example cells of different types suggest that this feature effectively quantifies differences in the fine scale synaptic distribution of different cell types (Fig. 4c).</p>
<p>Consistent with that, the bipolar example shown here contains an elevated number of such small protrusions, including in the somatic region, whereas the smoother dendritic shafts of this basket cell example contain virtually no counts in these bins. (Fig. 4c).</p>
<p>This distance dependent PSS histogram was calculated for all neurons within the column. We then recomputed a low dimensional embedding by aggregating nucleus, somatic and PSS features (Fig. 4d).</p>
</div>
</div>
</div>
<div class="column-page">
<p><img src="PSS_diagram_Figure4.png" class="img-fluid"></p>
<p>Post Synaptic Shape (PSS) Features. a) Procedure for building a PSS dictionary model. The set of shapes is used to train a PointNet autoencoder which learns a latent feature vector of a fixed size (1024). This autoencoder is then applied to all shapes in the dictionary to generate a set of latent feature vectors. K-means with K = 30 is applied to this to obtain a set of cluster centers for binning the shapes. b) Schematic showing procedure for creating the PSS histogram. The PSS are binned by shape type (30 bins calculated from a) and distance from the soma (4 bins) from 0 - 60 microns with 15 micron bin sizes. For each synapse, its shape type is determined by finding the closest cluster center in the latent feature space and its distance bin is determined by using the location of the synapse center. The resulting histogram is a 2D histogram shown above with the shapes in the x direction and distances in the y direction. c) Examples of 60 micron cutouts of 2 different types of cells with their spatial histograms shown as heatmaps. The top row shows the shape of the cluster center of each of the 30 clusters. In each heat map, darker boxes indicate higher values. Observe that the pattern of the overall histogram is very different between the two cell types. d) 2D UMAP of all neurons in the column inferred after concatenating nucleus, soma and PSS features. Observe the separation of inhibitory and excitatory neurons.</p>
</div>
</section>
<section id="classifiers" class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="classifiers">Classifiers</h4>
<p>For the cells in the column, we collected three sets of features: nucleus and soma features for all cells, and PSS features for the neurons. The cell classes and sub-classes defined across the column appear to have distinct fingerprints when visualized by the z-scored feature heatmaps (Fig. 5a). A low dimensional embedding of the combined features of neurons colored by sub-class supports that perspective (Fig. 5b). However, in order to quantitatively compare how well different feature spaces separate cells at different levels of distinction, we fit a collection of classification models that could be applied to all the cells in the dataset (see methods). We took a hierarchical approach (Fig. 5c), developing a cascade of classifiers to sort cells at increasingly finer distinctions. At each step (1-5) in the hierarchy we trained a series of classifiers using the different feature sets available for that step (Fig. 5d).</p>
<p>We then integrated the steps into a meta-model, where individual cells are sequentially sorted down the hierarchical tree. We found an optimal combination of classifiers which predicted cell types with a 92% overall accuracy (Fig. 5d) (see methods). This performance is high across different portions of the cell-type space, including between the inhibitory sub-classes whose definitions historically have come from their targeting patterns or dendritic morphology. The largest single confusion between types here was between adjacent layers of similar pyramidal classes, where arguably the strict laminar boundaries found in the manual labels might not be biologically accurate (Fig. 5e). This demonstrates that these features are indeed useful for separating cell-types based on local somatic reconstructions of cortical cells.</p>
<div class="callout-note callout callout-style-default no-icon callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-5-contents" aria-controls="callout-5" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Text References to Figure 5
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-5" class="callout-5-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The cell classes and sub-classes defined across the column appear to have distinct fingerprints when visualized by the z-scored feature heatmaps (Fig. 5a).</p>
<p>A low dimensional embedding of the combined features of neurons colored by sub-class supports that perspective (Fig. 5b).</p>
<p>We took a hierarchical approach (Fig. 5c), developing a cascade of classifiers to sort cells at increasingly finer distinctions.</p>
<p>At each step (1-5) in the hierarchy we trained a series of classifiers using the different feature sets available for that step (Fig. 5d).</p>
<p>We found an optimal combination of classifiers which predicted cell types with a 92% overall accuracy (Fig. 5d) (see methods).</p>
<p>The largest single confusion between types here was between adjacent layers of similar pyramidal classes, where arguably the strict laminar boundaries found in the manual labels might not be biologically accurate (Fig. 5e).</p>
</div>
</div>
</div>
<div class="column-page">
<p><img src="Minnie_Classifier_Figure5.png" class="img-fluid"></p>
<p>A combination of nucleus, soma, and PSS features predict neuronal and nonneuronal subclasses for unlabelled cells. ai) depth, nucleus and soma feature matrix depicting z score values for all cells within the manually labeled column. Cells ordered by manual subclass labels, class and subclass colors noted in the bars at the top. Some sub-classes are compressed relative to others in order to better illustrate the variation across classes. Dashed marks along the x axis denote segments of 100 cells (1127 excitatory neurons, 141 inhibitory neurons, 325 nonneurons) aii) pss feature matrix for all neurons within the manually labeled column. Cell ordering matches the matrix in ai. b) 2D UMAP embedding inferred from depth, nucleus, soma, and pss features of all neurons within the column dataset colored by human labeled subclass. c) diagram of hierarchical model framework to predict nonneuronal class and neuronal subclass using a set of 5 models. Initial objects are outputs of a segmentation based nucleus detection model (in preparation) and include neuronal nuclei, nonneuronal nuclei and errors. Cells are subsequently sorted and pushed through the hierarchical framework based on predictions from higher level models. d) Accuracy and F1 scores for all trained models at each level of the hierarchical framework in b. Bolded values denote which model was used in the metamodel due to highest performance. Overall metamodel performance based on the confusion matrix in c is noted in the lower right corner. e) Confusion matrix results for all cells within the column based on subclass predictions from the hierarchical metamodel.</p>
</div>
</section>
<section id="generalization-to-other-datasets" class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="generalization-to-other-datasets">Generalization to other datasets</h4>
<p>To test whether the features we extracted were generalizable to other datasets from cortex, the same metrics were collected from a second dataset (see methods). The second dataset covers all layers of cortex, but is only 50um thick and so most cells are severely cut off. The same nucleus, soma and PSS features were extracted for all cells. Cells in the second dataset occupy similar areas when transformed into the low embedding space fit by the features extracted from the first dataset (Fig 6b-d). Similar to the first dataset, as more features are added, the separation between cell class and neuronal subclass increases. Further, the feature to feature correlations between the soma and nucleus features are highly similar between the two datasets (Fig. 6e). Because the PSS feature used a 60um cutout the PSS spatial histograms were often highly truncated for some cells, causing systematic differences between the datasets. We could however, use a meta-model constructed solely with nucleus and somatic features to predict sub-class in the second dataset in a way that aligns with what we expected based upon performance tests in the larger dataset (Fig. 6f, Extended Data Fig. 1b). This suggests these features are generally useful and reliable metrics that can be used to identify cells in experiments that are not large enough to capture morphology beyond the soma.</p>
<div class="callout-note callout callout-style-default no-icon callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-6-contents" aria-controls="callout-6" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Text References to Figure 6
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-6" class="callout-6-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The second dataset covers all layers of cortex, but is only 50um thick and so most cells are severely cut off (Fig. 6a).</p>
<p>The same nucleus, soma and PSS features were extracted for all cells. Cells in the second dataset occupy similar areas when transformed into the low embedding space fit by the features extracted from the first dataset (Fig 6b-d).</p>
<p>Further, the feature to feature correlations between the soma and nucleus features are highly similar between the two datasets (Fig. 6e).</p>
<p>We could however, use a meta-model constructed solely with nucleus and somatic features to predict sub-class in the second dataset in a way that aligns with what we expected based upon performance tests in the larger dataset (Fig. 6f, Extended Data Fig. 1b).</p>
</div>
</div>
</div>
<div class="column-page">
<p><img src="Basil_Minnie_Basil_Figure6.png" class="img-fluid"></p>
<p>Application of soma and nucleus features to second dataset. A) An overview of the dataset that covers a mm2 of cortex but is only 50um thick. Rendering of all the nuclei analyzed within this dataset, colored by manual neuronal and nonneuronal class labels. B) A UMAP of the nucleus features from this dataset, colored by human labeled cell class. The UMAP is fit based on dataset 1, and the points from dataset 1 are included in this plot in gray. C) a UMAP of the nucleus and soma features, again colored by human labeled cell classes and dataset 1 points in gray. D) A UMAP plot of all features, where the neurons are colored by their class. E) The canonical correlation analysis of soma and nucleus features across the two datasets. F) To the left, a 2D UMAP embedding inferred from soma and nucleus features displaying all cells from the column in dataset 1, including errors. Colored by manual subclass labels. On the right, the same embedding as before except dataset 1 is in gray. Overlaid is dataset 2 now colored by metamodel subclass predictions.</p>
</div>
</section>
</section>
<section id="discussion" class="level3">
<h3 class="anchored" data-anchor-id="discussion">Discussion</h3>
<p>Here we report the first large scale quantitative study of the somatic region of cortical brain cells using electron microscopy. We’ve shown that different cell-types have systematic differences in their local somatic features that reveal interesting biological patterns and allow cell-classes to be quantitatively distinguished.</p>
<p>For example, nuclear infoldings have been observed in a wide variety of cell types, neuronal and non-neuronal, both in live cells and fixed material.@jorgens_deep_2017,<span class="citation" data-cites="shimojima_emerin_2017">Shimojima et al. (<a href="#ref-shimojima_emerin_2017" role="doc-biblioref">2017</a>)</span>,<span class="citation" data-cites="schoen_nanoscale_2017">Schoen et al. (<a href="#ref-schoen_nanoscale_2017" role="doc-biblioref">2017</a>)</span> In dataset 1, the degree of nuclear infoldings varied starkly by excitatory sub-class with different patterns of variation within laminar layers. Specifically, we report the existence of a spatial gradient within excitatory cells in layer 4, which might reflect previously described gradients in gene expression. <span class="citation" data-cites="tasic_shared_2018">Tasic et al. (<a href="#ref-tasic_shared_2018" role="doc-biblioref">2018</a>)</span> Alternatively, the overall pattern of nuclear infolding which is shared across layer 5 types, might suggest a common mechanism despite their broad differences in gene expression, electrophysiological properties and connectivity. <span class="citation" data-cites="gouwens_classification_2019">Gouwens et al. (<a href="#ref-gouwens_classification_2019" role="doc-biblioref">2019</a>)</span>,<span class="citation" data-cites="tasic_shared_2018">Tasic et al. (<a href="#ref-tasic_shared_2018" role="doc-biblioref">2018</a>)</span> These gradients in excitatory cells could be related to a developmental mechanism, given that cortex develops in a laminar fashion, while interneurons, which don’t exhibit laminar shifts in nuclear infolding, migrate in from other parts of the brain. 35</p>
<p>There is some evidence that in some cell types that nucleus infoldings are distinct calcium microdomains. <span class="citation" data-cites="turner_reconstruction_2022">Turner et al. (<a href="#ref-turner_reconstruction_2022" role="doc-biblioref">2022</a>)</span> However, their functional importance has not been clearly established. Infolding of the nuclear membrane should reduce the intranuclear diffusion time to the nuclear membrane and increase the total surface area. A natural hypothesis would thus predict that cells with a higher demand for producing RNA out of the nucleus, should therefore have more infolding. However, our observations do not support this hypothesis because all subclasses of layer 5 excitatory cells (which span a wide range of sizes and overall levels of gene expression) have remarkably consistent levels of infolding. On the other hand, we observe a remarkably consistent scaling of the volume of the nucleus and the volume of the cytoplasm of cells across these subclasses, which supports the view that cells scale their overall size in relation to demands of gene expression.</p>
<p>A mechanism of infolding has been proposed whereby laminar associated domains (LADs) on the heterochromatin attach to the nuclear envelope, and a force is exerted that brings the heterochromatin under tension which causes the membrane to fold. <span class="citation" data-cites="schoen_nanoscale_2017">Schoen et al. (<a href="#ref-schoen_nanoscale_2017" role="doc-biblioref">2017</a>)</span> If LAD accessibility and binding to the lamin is driving nuclear infolding, then these data would suggest that measurements of these phenomena should exhibit clear spatial patterns across the cortical depth. Further, genomic analyses should show consistency of LAD accessibility across the layer 5 pyramidal sub-classes, despite large differences in gene expression. Examining the developmental origins of infolding within these varied layer 5 types could produce insight into the mechanisms underlying this phenomena.</p>
<p>Our results reveal some novel properties of distinct cell classes. Near projecting neurons were first described as a set of Layer 5 cells expressing Slc17a8, which did not project to contralateral cortex or any of several sub-cortical structures. <span class="citation" data-cites="sorensen_correlated_2015">Sorensen et al. (<a href="#ref-sorensen_correlated_2015" role="doc-biblioref">2015</a>)</span> They have also been characterized as being enriched in a genetic line, and having a small round somatic shape. <span class="citation" data-cites="kim_three_2015">E. J. Kim et al. (<a href="#ref-kim_three_2015" role="doc-biblioref">2015</a>)</span>,<span class="citation" data-cites="sorensen_correlated_2015">Sorensen et al. (<a href="#ref-sorensen_correlated_2015" role="doc-biblioref">2015</a>)</span> Since then, transcriptomics and retro-seq experiments has confirmed4 that they are a rare, but highly conserved subclass of neurons across several species. <span class="citation" data-cites="hodge_conserved_2019">Hodge et al. (<a href="#ref-hodge_conserved_2019" role="doc-biblioref">2019</a>)</span> Morphologically they have been characterized as having sparse long straight basal dendrites and distinct electrophysiological properties.8 Our description here adds some important facts to their functional profile. They appear to have one of the lowest densities of synaptic input of all cell sub-classes we identified, on both shafts, somas and spines. What synapses they do have on spines are often on uncharacteristically long and spindle-like spines. This suggests that NP cells might be an intriguing target to understand the role of spine shape in cortical computation. <span class="citation" data-cites="harnett_synaptic_2012">Harnett et al. (<a href="#ref-harnett_synaptic_2012" role="doc-biblioref">2012</a>)</span>, 39, <span class="citation" data-cites="cornejo_voltage_2022">Cornejo, Ofer, and Yuste (<a href="#ref-cornejo_voltage_2022" role="doc-biblioref">2022</a>)</span></p>
<p>Our observation that spine shapes vary systematically across cell types could relate to differences in the average age or nature of spines across those types, <span class="citation" data-cites="knott_spine_2006">Knott et al. (<a href="#ref-knott_spine_2006" role="doc-biblioref">2006</a>)</span> as at least VIP dendrites have been shown to have much more motile spines than other interneurons, <span class="citation" data-cites="georgiou_subpopulation_2022">Georgiou et al. (<a href="#ref-georgiou_subpopulation_2022" role="doc-biblioref">2022</a>)</span> and some spiny SST+ interneurons have highly multi-synaptic spine like protrusions.23 We observed some clear but previously unquantified differences in features of different inhibitory sub-classes in cortex. Basket cells are known to have a high rate of connectivity to pyramidal cells and vice versa, on a per cell basis, but our data shows that this correlates with a higher overall density of synapses onto those cells compared to other subtypes (Extended Data Fig 5). <span class="citation" data-cites="campagnola_local_2022">Campagnola et al. (<a href="#ref-campagnola_local_2022" role="doc-biblioref">2022</a>)</span>,<span class="citation" data-cites="jiang_principles_2015">Jiang et al. (<a href="#ref-jiang_principles_2015" role="doc-biblioref">2015</a>)</span>,<span class="citation" data-cites="hioki_cell_2013">Hioki et al. (<a href="#ref-hioki_cell_2013" role="doc-biblioref">2013</a>)</span> Furthermore, the martinotti subtypes have some notable variation of synaptic density on their soma, and neurogliaform cells have a strikingly lower density of synapses on their somatic region.</p>
<p>It should be noted explicitly that the level of description of cell classes and sub-classes presented here is not meant to reflect a final refined definition of cell types in the mouse visual cortex. In particular, the 100 um column is small enough that it doesn’t fully sample the space of inhibitory cells. All the inhibitory cells were not given a label and some known types are missing or only single examples are found. For example, there are no chandelier cells found in that dataset. Exploring the space of soma and nucleus features often co-varies with variation in connectivity patterns of individual cells, suggesting that further sub-divisions are necessary and justified by these data. Furthermore, the precise sub-class definitions of individual excitatory cells is subject to reasonable debate. In particular, classifying excitatory cells across the layer ⅔ and layer 4, and layer 5 and layer 6 borders can vary from rater to rater on a 10-20 microns length scale. Nonetheless, the variation across the major classes and sub-classes shown here demonstrates that the somatic structure of individual neurons is modulated by cell type specialization.</p>
<p>The classifications we have produced here will be immensely useful for researchers mining these data for biologically interesting patterns of connectivity. Because the features used in this study do not require immense proofreading, the predictions are accurate across most of the dataset. This means the cell type connectivity of individual neurons or axonal fragments can be directly queried for targets which contain a single soma in the dataset. In the present release of the dataset, this means over 50% of a typical cell’s outputs can be classified. <span class="citation" data-cites="harnett_synaptic_2012">Harnett et al. (<a href="#ref-harnett_synaptic_2012" role="doc-biblioref">2012</a>)</span> Another use case for these predictions is to aggregate common inputs to a subclass of cells to find axon fragments with strong specificity for that subclass. Furthermore, these features can be extracted from datasets where full reconstruction is not possible, as we demonstrated with Dataset 2. These use cases demonstrate how an accurate data set wide cell classification transforms such datasets into a research resource for exploring the cell type specificity of connectivity. This success reinforces a more general phenomenon, that not only are different cell types expressing different proteins, but that these changes leave a unique imprint on cells that can be measured in highly localized structures with electron microscopy.</p>
</section>
<section id="methods" class="level3">
<h3 class="anchored" data-anchor-id="methods">Methods</h3>
<section id="dataset-1" class="level5">
<h5 class="anchored" data-anchor-id="dataset-1">Dataset 1</h5>
<p>Dataset 1 consists of a 1.4mm x 800um x 800um volumetric serial section EM dataset from mouse visual cortex of a male P87 mouse. This covers all layers of cortex and spanning primary visual cortex and two higher visual areas. The dataset has been described in detail elsewhere. <span class="citation" data-cites="kim_implications_2021">J.-Y. Kim and Paredes (<a href="#ref-kim_implications_2021" role="doc-biblioref">2021</a>)</span> Briefly, two photon imaging was performed on the mouse, which was subsequently prepared for electron microscopy. The specimen was then sectioned and imaged using transmission electron microscopy. <span class="citation" data-cites="sorensen_correlated_2015">Sorensen et al. (<a href="#ref-sorensen_correlated_2015" role="doc-biblioref">2015</a>)</span> The images were then stitched, aligned,7 and processed through a deep learning segmentation algorithm, <span class="citation" data-cites="markram_reconstruction_2015">Markram et al. (<a href="#ref-markram_reconstruction_2015" role="doc-biblioref">2015</a>)</span> followed by manual proofreading (manuscript in preparation). For dataset 1, all cells within a 100um region were labeled by human experts as a nonneuron, neuron or error with class and subclass labels where applicable. This is described in detail elsewhere (manuscript in preparation). Cells where the nucleus was surrounded by error ID 0 were excluded to remove edge volume edge effects. Due to high levels of proofreading in the column, there were very few errors thus the training set was augmented with manually labeled errors from the entire dataset.</p>
</section>
<section id="dataset-2" class="level5">
<h5 class="anchored" data-anchor-id="dataset-2">Dataset 2</h5>
<p>The second dataset covers a millimeter square cross-sectional area, and 40 microns of depth within primary visual cortex of a P49 male mouse. This dataset is described in detail elsewhere (manuscript in preparation). The largest available segmentation spans Layer 2/3 of cortex through to Layer 6. After applying the nuclear detection model (manuscript in preparation) and filtering out all nuclear objects below 25 um3 and cells that were cut off by the volume border, 86,91648 cells were used for the analysis. Class type of each cell was labeled manually and used as ground truth. Due to thinness of the volume, much of the distal cell morphologies were cut off and thus subclass type labeling was not possible. Nuclear and somatic meshes were cleaned using the same heuristic procedure outlined above.</p>
</section>
<section id="generating-nucleus-features" class="level5">
<h5 class="anchored" data-anchor-id="generating-nucleus-features">Generating Nucleus Features</h5>
<p>We analyzed nuclei using the results of a deep neural network segmentation (manuscript in preparation), extracted the mesh using marching cubes and obtained the largest component of the detected mesh.</p>
<p>We further filtered the detections from this algorithm in that any detected objects less than 25 um3 were filtered out as errors. Nuclear features were then extracted on the remaining meshes (n=129,769). These features included, nucleus volume, nucleus area, the area to volume ratio, nucleus surface area within an infolding, the fraction of the total surface area within an infolding, and cortical depth (measured as the distance from the pial surface). Nucleus fold features were extracted by creating a shrink wrapped37 mesh for each nucleus mesh. We then calculated the distance of each vertex on the nucleus mesh from the shrink-wrapped mesh and any vertex further than 150 nm was considered within an infolding.</p>
</section>
<section id="generating-somatic-features" class="level5">
<h5 class="anchored" data-anchor-id="generating-somatic-features">Generating Somatic Features</h5>
<p>For each nucleus detection the somatic compartment was identified as the ID in the segmentation which surrounded &gt;80% of the nucleus. Somatic segmentations went through a heuristic cleaning procedure to remove missing slices of data and incorrectly merged fragments. Since each soma was matched to its corresponding nucleus, 15 microns surrounding the nucleus center of mass was cut out from the dense segmentation and converted into a binary mask. Binary dilation by 5 voxels in 3d was performed, followed by filling of all holes, and then binary erosion of 3 voxels. The resulting binary mask was meshed using marching cubes and connected component analysis was run on the result. The largest connected component mesh was retained, and any disconnected components were dropped. Somatic features were extracted for all 129,769 nuclear detections within the dataset. These somatic features included soma area, soma volume, the area to volume ratio, the number of synapses on the somatic cutout, and the soma synapse density. Using both the somatic and nucleus meshes, we calculated the ratio between the nucleus volume and soma volume and the offset between the two, measured as the euclidean distance between nuclear center of mass and soma center of mass.</p>
<p>Filtering procedure: After identifying the segment IDs within a 15um bounding box around each nucleus, if over 20% of these IDs corresponded to error ID 0, they were filtered out. The majority of these error cases were cells close to the volume border or areas in the volume with higher segmentation errors such as those near blood vessels. Somatic meshes that were merged with other somas in the dataset were also filtered out. These cells were statistical outliers that clustered together in the low embedding feature space based on soma and nucleus features (Extended Data Fig. 1b) and thus were removed based on their location within that space. Finally, cells that were predicted as errors based on the soma and nucleus feature object model were also removed from analysis. This resulted in a final set of 86,916,454 cells, neuronal and nonneuronal.</p>
</section>
<section id="generating-pss-features" class="level5">
<h5 class="anchored" data-anchor-id="generating-pss-features">Generating PSS Features</h5>
<p>Around each synapse, we extracted a 3500nm region to obtain the synapse region mesh. This mesh was then segmented using the CGAL surface segmentation algorithm38 which splits regions based on differences in thickness. We adapted our previously developed method24 to identify the PSS region by using a local skeleton calculated from the synapse region mesh, rather than a precomputed whole cell mesh. This allowed us to adapt this method for cells in the dataset without the need for proofreading.</p>
<p>Given a cell for which all PSS have been extracted within a 60 micron radius from the nucleus center, the objective was to build a descriptor that encapsulates the various properties of the PSS. In particular, we aim to capture two of these properties: the type of shape of the PSS and the distance of the PSS from the soma. For the shape, a dictionary of all shape types is built using the dictionary dataset from. <span class="citation" data-cites="campagnola_local_2022">Campagnola et al. (<a href="#ref-campagnola_local_2022" role="doc-biblioref">2022</a>)</span> These shapes were rotationally normalized and used to train a pointnet autoencoder39 to learn a latent representation of size 1024. The high dimensional latent space spanning all these shapes is a continuous space (Extended Data Fig 3) which was used to generate a Bag of Words model40 for the shapes. For this, we performed K-means clustering with K=30 to estimate cluster centers. The top row of Fig. 4c shows the shape in the dictionary that is closest to each of these cluster centers. For distance binning, we split the 60 micron radius around the nucleus center into four 15 micron radial bins (Figure 4b). All PSS were then binned according to their shape and distance properties.</p>
</section>
<section id="hierarchical-framework" class="level5">
<h5 class="anchored" data-anchor-id="hierarchical-framework">Hierarchical Framework</h5>
<p>We defined an object as the segmentation associated with a predicted nucleus (manuscript in preparation) from which nucleus, soma, and PSS features could be extracted. A hierarchical framework was designed to predict the cell type of any such object (Fig. 5c). From Dataset 1, there were 86,916objects. The first level in the hierarchy predicts whether the object was a neuron, nonneuron, or an error. All objects predicted as errors were excluded from all subsequent analyses except for metamodel prediction evaluation. Nonneuronal cells were then classified as one of the following: astrocyte, microglia, oligodendrocyte, oligodendrocyte precursor cells (OPC), or pericyte. For neurons, cells were predicted as either excitatory or inhibitory followed by a separate subclass classifier for each class type. Excitatory subclasses: Layer 2/3 pyramidal, Layer 4 pyramidal, Layer 5 IT, Layer 5 ET, Layer 5 near projecting (NP) pyramidal, Layer 6 IT, Layer 6 CT pyramidal. Inhibitory subclasses: Basket cells, Bipolar cells, Martinotti cells, and Neurogliaform cells.</p>
<p>Soma and nucleus features were extracted from the 3D mesh of all objects and PSS features were extracted from all neurons within the column. For each level of the hierarchy, multiple classifiers were trained using either nucleus and depth features only, nucleus depth and soma features, or nucleus soma depth and PSS features. Within each level, performance of the various classifiers was evaluated based on accuracy and F1 score (a measure for precision and recall). The metamodel was defined as the sequential combination of the best performing classifiers at each level.</p>
</section>
<section id="individual-classifiers" class="level5">
<h5 class="anchored" data-anchor-id="individual-classifiers">Individual Classifiers</h5>
<p>For each classifier, model type was chosen using a randomized grid search for the following models: Support Vector Machine SVM with a linear kernel, SVM with a radial basis function kernel, Nearest Neighbors, Random Forest Classifier, Decision Tree and Neural Network. For each type, 50 models were trained with varying parameters and the top performing model was chosen. This model was then further optimized using 5-fold cross validation with 20% held out for validation. Training, and test examples were held consistent across models for direct performance comparison within each level.</p>
<p>The top level of the hierarchy (the object model), distinguished neurons from non-neurons as well as erroneous detections. Here, the nucleus features alone were sufficient to separate neuronal from non-neuronal nuclei as well as erroneous detections with an F1 score of 98.7%. Adding somatic features to this model did not significantly change performance. For the non-neuronal class model,, somatic features did notably improve the accuracy of distinguishing between non-neuronal types from 84% to 95%. The neuron class model simply distinguished excitatory from inhibitory neurons. Here, again the soma features improved the classification accuracy over the nucleus only features from 88% to 98%. Consistent with the qualitative impression given by the low dimensional projections, the principle confusion for the soma-nucleus models exists between the inhibitory and excitatory cells, particularly in Layer 5 (Extended Data Fig. 4). Adding in the PSS features raised performance to 99.6%. The excitatory subclass model distinguished between 7 different laminar specific pyramidal types. Here, the soma and nucleus features performed the best, with 90% accuracy. The inhibitory subclass model distinguished between the 4 sub-classes of inhibitory cells found in the columnar region. Performance improves significantly as you add in the somatic and PSS features, topping out with an F1 score of 93.1%.</p>
</section>
<section id="metamodel" class="level5">
<h5 class="anchored" data-anchor-id="metamodel">Metamodel</h5>
<p>We tested the overall performance for predicting the leaves of the tree using different combinations of the individual classifiers. We found that the best performing meta-model used the best performing classifier at each step (bolded Fig. 5d), with the exception of one, the object class step. This was because the classifier trained on nucleus features alone had a larger number of confusions between neurons and non-neurons than the classifier trained with soma and nucleus features, leading to lower overall performance (accuracy &amp; F1) in the meta-model.</p>
</section>
<section id="contributions" class="level5">
<h5 class="anchored" data-anchor-id="contributions">Contributions</h5>
<p>Analyzed Data: LE, SS, FC, CSM Developed Nucleus Model: SM, GM, LE, FC Cell Typing: AB, NDC, CSM Paper Writing: LE, SS, FC Dataset + segmentation generation: collaboration Proofreading: collaboration Analysis infrastructure: collaboration</p>
</section>
</section>
<section id="references" class="level1">




</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-ascoli_petilla_2008" class="csl-entry" role="doc-biblioentry">
Ascoli, Giorgio A., Lidia Alonso-Nanclares, Stewart A. Anderson, German Barrionuevo, Ruth Benavides-Piccione, Andreas Burkhalter, György Buzsáki, et al. 2008. <span>“Petilla Terminology: Nomenclature of Features of <span>GABAergic</span> Interneurons of the Cerebral Cortex.”</span> <em>Nature Reviews Neuroscience</em> 9 (7): 557–68. <a href="https://doi.org/10.1038/nrn2402">https://doi.org/10.1038/nrn2402</a>.
</div>
<div id="ref-campagnola_local_2022" class="csl-entry" role="doc-biblioentry">
Campagnola, Luke, Stephanie C. Seeman, Thomas Chartrand, Lisa Kim, Alex Hoggarth, Clare Gamlin, Shinya Ito, et al. 2022. <span>“Local Connectivity and Synaptic Dynamics in Mouse and Human Neocortex.”</span> <em>Science</em> 375 (6585): eabj5861. <a href="https://doi.org/10.1126/science.abj5861">https://doi.org/10.1126/science.abj5861</a>.
</div>
<div id="ref-consortium_functional_2021" class="csl-entry" role="doc-biblioentry">
Consortium, MICrONS, J. Alexander Bae, Mahaly Baptiste, Agnes L. Bodor, Derrick Brittain, JoAnn Buchanan, Daniel J. Bumbarger, et al. 2021. <span>“Functional Connectomics Spanning Multiple Areas of Mouse Visual Cortex.”</span> <span>bioRxiv</span>. <a href="https://doi.org/10.1101/2021.07.28.454025">https://doi.org/10.1101/2021.07.28.454025</a>.
</div>
<div id="ref-cornejo_voltage_2022" class="csl-entry" role="doc-biblioentry">
Cornejo, Victor Hugo, Netanel Ofer, and Rafael Yuste. 2022. <span>“Voltage Compartmentalization in Dendritic Spines in Vivo.”</span> <em>Science</em> 375 (6576): 82–86. <a href="https://doi.org/10.1126/science.abg0501">https://doi.org/10.1126/science.abg0501</a>.
</div>
<div id="ref-davis_microcircuitry_1979" class="csl-entry" role="doc-biblioentry">
Davis, T. L., and P. Sterling. 1979. <span>“Microcircuitry of Cat Visual Cortex: Classification of Neurons in Layer <span>IV</span> of Area 17, and Identification of the Patterns of Lateral Geniculate Input.”</span> <em>The Journal of Comparative Neurology</em> 188 (4): 599–627. <a href="https://doi.org/10.1002/cne.901880407">https://doi.org/10.1002/cne.901880407</a>.
</div>
<div id="ref-delgado-escueta_fine_1992" class="csl-entry" role="doc-biblioentry">
Delgado-Escueta, Antonio V. 1992. <span>“The Fine Structure of the Nervous System: Neurons and Their Supporting Cells. Alan Peters , Sanford l. Palay , Henry de f. Webster.”</span> <em>The Quarterly Review of Biology</em> 67 (1): 80–80. <a href="https://doi.org/10.1086/417517">https://doi.org/10.1086/417517</a>.
</div>
<div id="ref-georgiou_subpopulation_2022" class="csl-entry" role="doc-biblioentry">
Georgiou, Christina, Vassilis Kehayas, Kok Sin Lee, Federico Brandalise, Daniela A. Sahlender, Jerome Blanc, Graham Knott, and Anthony Holtmaat. 2022. <span>“A Subpopulation of Cortical <span>VIP</span>-Expressing Interneurons with Highly Dynamic Spines.”</span> <em>Communications Biology</em> 5 (1): 1–15. <a href="https://doi.org/10.1038/s42003-022-03278-z">https://doi.org/10.1038/s42003-022-03278-z</a>.
</div>
<div id="ref-gilbert_morphology_1979" class="csl-entry" role="doc-biblioentry">
Gilbert, Charles D., and Torsten N. Wiesel. 1979. <span>“Morphology and Intracortical Projections of Functionally Characterised Neurones in the Cat Visual Cortex.”</span> <em>Nature</em> 280 (5718): 120–25. <a href="https://doi.org/10.1038/280120a0">https://doi.org/10.1038/280120a0</a>.
</div>
<div id="ref-gouwens_classification_2019" class="csl-entry" role="doc-biblioentry">
Gouwens, Nathan W., Staci A. Sorensen, Jim Berg, Changkyu Lee, Tim Jarsky, Jonathan Ting, Susan M. Sunkin, et al. 2019. <span>“Classification of Electrophysiological and Morphological Neuron Types in the Mouse Visual Cortex.”</span> <em>Nature Neuroscience</em> 22 (7): 1182–95. <a href="https://doi.org/10.1038/s41593-019-0417-0">https://doi.org/10.1038/s41593-019-0417-0</a>.
</div>
<div id="ref-hage_synaptic_2022" class="csl-entry" role="doc-biblioentry">
Hage, Travis A, Alice Bosma-Moody, Christopher A Baker, Megan B Kratz, Luke Campagnola, Tim Jarsky, Hongkui Zeng, and Gabe J Murphy. 2022. <span>“Synaptic Connectivity to L2/3 of Primary Visual Cortex Measured by Two-Photon Optogenetic Stimulation.”</span> Edited by Tianyi Mao and Ronald L Calabrese. <em><span>eLife</span></em> 11 (January): e71103. <a href="https://doi.org/10.7554/eLife.71103">https://doi.org/10.7554/eLife.71103</a>.
</div>
<div id="ref-harnett_synaptic_2012" class="csl-entry" role="doc-biblioentry">
Harnett, Mark T., Judit K. Makara, Nelson Spruston, William L. Kath, and Jeffrey C. Magee. 2012. <span>“Synaptic Amplification by Dendritic Spines Enhances Input Cooperativity.”</span> <em>Nature</em> 491 (7425): 599–602. <a href="https://doi.org/10.1038/nature11554">https://doi.org/10.1038/nature11554</a>.
</div>
<div id="ref-hioki_cell_2013" class="csl-entry" role="doc-biblioentry">
Hioki, Hiroyuki, Shinichiro Okamoto, Michiteru Konno, Hiroshi Kameda, Jaerin Sohn, Eriko Kuramoto, Fumino Fujiyama, and Takeshi Kaneko. 2013. <span>“Cell Type-Specific Inhibitory Inputs to Dendritic and Somatic Compartments of Parvalbumin-Expressing Neocortical Interneuron.”</span> <em>Journal of Neuroscience</em> 33 (2): 544–55. <a href="https://doi.org/10.1523/JNEUROSCI.2255-12.2013">https://doi.org/10.1523/JNEUROSCI.2255-12.2013</a>.
</div>
<div id="ref-hodge_conserved_2019" class="csl-entry" role="doc-biblioentry">
Hodge, Rebecca D, Trygve E Bakken, Jeremy A Miller, Kimberly A Smith, Eliza R Barkan, Lucas T Graybuck, Jennie L Close, et al. 2019. <span>“Conserved Cell Types with Divergent Features in Human Versus Mouse Cortex.”</span> <em>Nature</em> 573 (7772): 61–68. <a href="https://doi.org/10.1038/s41586-019-1506-7">https://doi.org/10.1038/s41586-019-1506-7</a>.
</div>
<div id="ref-hwang_3d_2021" class="csl-entry" role="doc-biblioentry">
Hwang, Yang-Sun, Catherine Maclachlan, Jérôme Blanc, Anaëlle Dubois, Carl C H Petersen, Graham Knott, and Seung-Hee Lee. 2021. <span>“3D Ultrastructure of Synaptic Inputs to Distinct <span>GABAergic</span> Neurons in the Mouse Primary Visual Cortex.”</span> <em>Cerebral Cortex</em> 31 (5): 2610–24. <a href="https://doi.org/10.1093/cercor/bhaa378">https://doi.org/10.1093/cercor/bhaa378</a>.
</div>
<div id="ref-jiang_principles_2015" class="csl-entry" role="doc-biblioentry">
Jiang, Xiaolong, Shan Shen, Cathryn R. Cadwell, Philipp Berens, Fabian Sinz, Alexander S. Ecker, Saumil Patel, and Andreas S. Tolias. 2015. <span>“Principles of Connectivity Among Morphologically Defined Cell Types in Adult Neocortex.”</span> <em>Science</em> 350 (6264): aac9462. <a href="https://doi.org/10.1126/science.aac9462">https://doi.org/10.1126/science.aac9462</a>.
</div>
<div id="ref-jones_varieties_1975" class="csl-entry" role="doc-biblioentry">
Jones, E. G. 1975. <span>“Varieties and Distribution of Non-Pyramidal Cells in the Somatic Sensory Cortex of the Squirrel Monkey.”</span> <em>The Journal of Comparative Neurology</em> 160 (2): 205–67. <a href="https://doi.org/10.1002/cne.901600204">https://doi.org/10.1002/cne.901600204</a>.
</div>
<div id="ref-jorgens_deep_2017" class="csl-entry" role="doc-biblioentry">
Jorgens, Danielle M., Jamie L. Inman, Michal Wojcik, Claire Robertson, Hildur Palsdottir, Wen-Ting Tsai, Haina Huang, et al. 2017. <span>“Deep Nuclear Invaginations Are Linked to Cytoskeletal Filaments – Integrated Bioimaging of Epithelial Cells in 3D Culture.”</span> Edited by Andrew Ewald. <em>Journal of Cell Science</em> 130 (1): 177–89. <a href="https://doi.org/10.1242/jcs.190967">https://doi.org/10.1242/jcs.190967</a>.
</div>
<div id="ref-kanari_objective_2019" class="csl-entry" role="doc-biblioentry">
Kanari, Lida, Srikanth Ramaswamy, Ying Shi, Sebastien Morand, Julie Meystre, Rodrigo Perin, Marwan Abdellah, Yun Wang, Kathryn Hess, and Henry Markram. 2019. <span>“Objective Morphological Classification of Neocortical Pyramidal Cells.”</span> <em>Cerebral Cortex</em> 29 (4): 1719–35. <a href="https://doi.org/10.1093/cercor/bhy339">https://doi.org/10.1093/cercor/bhy339</a>.
</div>
<div id="ref-kim_three_2015" class="csl-entry" role="doc-biblioentry">
Kim, Euiseok J., Ashley L. Juavinett, Espoir M. Kyubwa, Matthew W. Jacobs, and Edward M. Callaway. 2015. <span>“Three Types of Cortical Layer 5 Neurons That Differ in Brain-Wide Connectivity and Function.”</span> <em>Neuron</em> 88 (6): 1253–67. <a href="https://doi.org/10.1016/j.neuron.2015.11.002">https://doi.org/10.1016/j.neuron.2015.11.002</a>.
</div>
<div id="ref-kim_implications_2021" class="csl-entry" role="doc-biblioentry">
Kim, Jae-Yeon, and Mercedes F. Paredes. 2021. <span>“Implications of Extended Inhibitory Neuron Development.”</span> <em>International Journal of Molecular Sciences</em> 22 (10): 5113. <a href="https://doi.org/10.3390/ijms22105113">https://doi.org/10.3390/ijms22105113</a>.
</div>
<div id="ref-knott_spine_2006" class="csl-entry" role="doc-biblioentry">
Knott, Graham W., Anthony Holtmaat, Linda Wilbrecht, Egbert Welker, and Karel Svoboda. 2006. <span>“Spine Growth Precedes Synapse Formation in the Adult Neocortex in Vivo.”</span> <em>Nature Neuroscience</em> 9 (9): 1117–24. <a href="https://doi.org/10.1038/nn1747">https://doi.org/10.1038/nn1747</a>.
</div>
<div id="ref-macrina_petascale_2021" class="csl-entry" role="doc-biblioentry">
Macrina, Thomas, Kisuk Lee, Ran Lu, Nicholas L. Turner, Jingpeng Wu, Sergiy Popovych, William Silversmith, et al. 2021. <span>“Petascale Neural Circuit Reconstruction: Automated Methods.”</span> <span>bioRxiv</span>. <a href="https://doi.org/10.1101/2021.08.04.455162">https://doi.org/10.1101/2021.08.04.455162</a>.
</div>
<div id="ref-markram_reconstruction_2015" class="csl-entry" role="doc-biblioentry">
Markram, Henry, Eilif Muller, Srikanth Ramaswamy, Michael W. Reimann, Marwan Abdellah, Carlos Aguado Sanchez, Anastasia Ailamaki, et al. 2015. <span>“Reconstruction and Simulation of Neocortical Microcircuitry.”</span> <em>Cell</em> 163 (2): 456–92. <a href="https://doi.org/10.1016/j.cell.2015.09.029">https://doi.org/10.1016/j.cell.2015.09.029</a>.
</div>
<div id="ref-martin_form_1984" class="csl-entry" role="doc-biblioentry">
Martin, K A, and D Whitteridge. 1984. <span>“Form, Function and Intracortical Projections of Spiny Neurones in the Striate Visual Cortex of the Cat.”</span> <em>The Journal of Physiology</em> 353 (August): 463–504. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1193318/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1193318/</a>.
</div>
<div id="ref-motta_dense_2019" class="csl-entry" role="doc-biblioentry">
Motta, Alessandro, Manuel Berning, Kevin M. Boergens, Benedikt Staffler, Marcel Beining, Sahil Loomba, Philipp Hennig, Heiko Wissler, and Moritz Helmstaedter. 2019. <span>“Dense Connectomic Reconstruction in Layer 4 of the Somatosensory Cortex.”</span> <em>Science</em> 366 (6469): eaay3134. <a href="https://doi.org/10.1126/science.aay3134">https://doi.org/10.1126/science.aay3134</a>.
</div>
<div id="ref-oberlaender_cell_2012" class="csl-entry" role="doc-biblioentry">
Oberlaender, Marcel, Christiaan P. J. de Kock, Randy M. Bruno, Alejandro Ramirez, Hanno S. Meyer, Vincent J. Dercksen, Moritz Helmstaedter, and Bert Sakmann. 2012. <span>“Cell Type-Specific Three-Dimensional Structure of Thalamocortical Circuits in a Column of Rat Vibrissal Cortex.”</span> <em>Cerebral Cortex (New York, N.Y.: 1991)</em> 22 (10): 2375–91. <a href="https://doi.org/10.1093/cercor/bhr317">https://doi.org/10.1093/cercor/bhr317</a>.
</div>
<div id="ref-pchitskaya_dendritic_2020" class="csl-entry" role="doc-biblioentry">
Pchitskaya, Ekaterina, and Ilya Bezprozvanny. 2020. <span>“Dendritic Spines Shape Analysis—Classification or Clusterization? Perspective.”</span> <em>Frontiers in Synaptic Neuroscience</em> 12. <a href="https://www.frontiersin.org/articles/10.3389/fnsyn.2020.00031">https://www.frontiersin.org/articles/10.3389/fnsyn.2020.00031</a>.
</div>
<div id="ref-peters_neuronal_1985" class="csl-entry" role="doc-biblioentry">
Peters, A., and D. A. Kara. 1985. <span>“The Neuronal Composition of Area 17 of Rat Visual Cortex. I. The Pyramidal Cells.”</span> <em>The Journal of Comparative Neurology</em> 234 (2): 218–41. <a href="https://doi.org/10.1002/cne.902340208">https://doi.org/10.1002/cne.902340208</a>.
</div>
<div id="ref-scala_phenotypic_2021" class="csl-entry" role="doc-biblioentry">
Scala, Federico, Dmitry Kobak, Matteo Bernabucci, Yves Bernaerts, Cathryn René Cadwell, Jesus Ramon Castro, Leonard Hartmanis, et al. 2021. <span>“Phenotypic Variation of Transcriptomic Cell Types in Mouse Motor Cortex.”</span> <em>Nature</em> 598 (7879): 144–50. <a href="https://doi.org/10.1038/s41586-020-2907-3">https://doi.org/10.1038/s41586-020-2907-3</a>.
</div>
<div id="ref-schoen_nanoscale_2017" class="csl-entry" role="doc-biblioentry">
Schoen, Ingmar, Lina Aires, Jonas Ries, and Viola Vogel. 2017. <span>“Nanoscale Invaginations of the Nuclear Envelope: Shedding New Light on Wormholes with Elusive Function.”</span> <em>Nucleus</em> 8 (5): 506–14. <a href="https://doi.org/10.1080/19491034.2017.1337621">https://doi.org/10.1080/19491034.2017.1337621</a>.
</div>
<div id="ref-scorcioni_l-measure_2008" class="csl-entry" role="doc-biblioentry">
Scorcioni, Ruggero, Sridevi Polavaram, and Giorgio A. Ascoli. 2008. <span>“L-Measure: A Web-Accessible Tool for the Analysis, Comparison and Search of Digital Reconstructions of Neuronal Morphologies.”</span> <em>Nature Protocols</em> 3 (5): 866–76. <a href="https://doi.org/10.1038/nprot.2008.51">https://doi.org/10.1038/nprot.2008.51</a>.
</div>
<div id="ref-shapson-coe_connectomic_2021" class="csl-entry" role="doc-biblioentry">
Shapson-Coe, Alexander, Michał Januszewski, Daniel R. Berger, Art Pope, Yuelong Wu, Tim Blakely, Richard L. Schalek, et al. 2021. <span>“A Connectomic Study of a Petascale Fragment of Human Cerebral Cortex.”</span> <span>bioRxiv</span>. <a href="https://doi.org/10.1101/2021.05.29.446289">https://doi.org/10.1101/2021.05.29.446289</a>.
</div>
<div id="ref-shimojima_emerin_2017" class="csl-entry" role="doc-biblioentry">
Shimojima, Masaya, Shinsuke Yuasa, Chikaaki Motoda, Gakuto Yozu, Toshihiro Nagai, Shogo Ito, Mark Lachmann, et al. 2017. <span>“Emerin Plays a Crucial Role in Nuclear Invagination and in the Nuclear Calcium Transient.”</span> <em>Scientific Reports</em> 7 (1): 44312. <a href="https://doi.org/10.1038/srep44312">https://doi.org/10.1038/srep44312</a>.
</div>
<div id="ref-sholl_dendritic_1953" class="csl-entry" role="doc-biblioentry">
Sholl, D. A. 1953. <span>“Dendritic Organization in the Neurons of the Visual and Motor Cortices of the Cat.”</span> <em>Journal of Anatomy</em> 87 (October): 387–406.1. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1244622/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1244622/</a>.
</div>
<div id="ref-sorensen_correlated_2015" class="csl-entry" role="doc-biblioentry">
Sorensen, Staci A., Amy Bernard, Vilas Menon, Joshua J. Royall, Katie J. Glattfelder, Tsega Desta, Karla Hirokawa, et al. 2015. <span>“Correlated Gene Expression and Target Specificity Demonstrate Excitatory Projection Neuron Diversity.”</span> <em>Cerebral Cortex</em> 25 (2): 433–49. <a href="https://doi.org/10.1093/cercor/bht243">https://doi.org/10.1093/cercor/bht243</a>.
</div>
<div id="ref-tasic_shared_2018" class="csl-entry" role="doc-biblioentry">
Tasic, Bosiljka, Zizhen Yao, Lucas T. Graybuck, Kimberly A. Smith, Thuc Nghi Nguyen, Darren Bertagnolli, Jeff Goldy, et al. 2018. <span>“Shared and Distinct Transcriptomic Cell Types Across Neocortical Areas.”</span> <em>Nature</em> 563 (7729): 72–78. <a href="https://doi.org/10.1038/s41586-018-0654-5">https://doi.org/10.1038/s41586-018-0654-5</a>.
</div>
<div id="ref-turner_reconstruction_2022" class="csl-entry" role="doc-biblioentry">
Turner, Nicholas L., Thomas Macrina, J. Alexander Bae, Runzhe Yang, Alyssa M. Wilson, Casey Schneider-Mizell, Kisuk Lee, et al. 2022. <span>“Reconstruction of Neocortex: Organelles, Compartments, Cells, Circuits, and Activity.”</span> <em>Cell</em> 185 (6): 1082–1100.e24. <a href="https://doi.org/10.1016/j.cell.2022.01.023">https://doi.org/10.1016/j.cell.2022.01.023</a>.
</div>
<div id="ref-wang_anatomical_2004" class="csl-entry" role="doc-biblioentry">
Wang, Yun, Maria Toledo-Rodriguez, Anirudh Gupta, Caizhi Wu, Gilad Silberberg, Junyi Luo, and Henry Markram. 2004. <span>“Anatomical, Physiological and Molecular Properties of Martinotti Cells in the Somatosensory Cortex of the Juvenile Rat.”</span> <em>The Journal of Physiology</em> 561 (1): 65–90. <a href="https://doi.org/10.1113/jphysiol.2004.073353">https://doi.org/10.1113/jphysiol.2004.073353</a>.
</div>
<div id="ref-wittmann_synaptic_2009" class="csl-entry" role="doc-biblioentry">
Wittmann, Malte, Gillian Queisser, Anja Eder, J. Simon Wiegert, C. Peter Bengtson, Andrea Hellwig, Gabriel Wittum, and Hilmar Bading. 2009. <span>“Synaptic Activity Induces Dramatic Changes in the Geometry of the Cell Nucleus: Interplay Between Nuclear Structure, Histone H3 Phosphorylation, and Nuclear Calcium Signaling.”</span> <em>Journal of Neuroscience</em> 29 (47): 14687–700. <a href="https://doi.org/10.1523/JNEUROSCI.1160-09.2009">https://doi.org/10.1523/JNEUROSCI.1160-09.2009</a>.
</div>
<div id="ref-zeisel_cell_2015" class="csl-entry" role="doc-biblioentry">
Zeisel, Amit, Ana B. Muñoz-Manchado, Simone Codeluppi, Peter Lönnerberg, Gioele La Manno, Anna Juréus, Sueli Marques, et al. 2015. <span>“Cell Types in the Mouse Cortex and Hippocampus Revealed by Single-Cell <span>RNA</span>-Seq.”</span> <em>Science</em> 347 (6226): 1138–42. <a href="https://doi.org/10.1126/science.aaa1934">https://doi.org/10.1126/science.aaa1934</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>